download_and_create_reference_datasets/v02/hail_scripts/write_combined_reference_data_ht.py

```diff
@@ -205,6 +206,28 @@ def custom_gnomad_select(ht):
                               0, ht.freq[ht.globals.freq_index_dict['gnomad_male']].AC)
     return selects
 
+
+def custom_gnomad_select_v3(ht):
+    """
+    Custom select for public gnomad dataset (which we did not generate). Extracts fields like
+    'AF', 'AN', and generates 'hemi'.
+    :param ht: hail table
+    :return: select expression dict
+    """
+    selects = {}
+    global_idx = hl.eval(ht.globals.freq_index_dict['adj'])
```

Comment:
@nawatts and @lfrancioli  can you check that these are correct with the v2 equivalent https://github.com/macarthur-lab/hail-elasticsearch-pipelines/pull/174/files#diff-35b65d916f1bb6cb19d59b94cde1478dR189